src/main/kotlin/ru/cib/clusterizer/controller/DockerClientController.kt

```diff
@@ -17,7 +17,7 @@ class DockerClientController(
     private val apiServices: Map<ConfigId, DockerApiService>
 ) {
 
-    @GetMapping("/client/{configId}/ping")
+    @GetMapping("/{configId}/client/ping")
```

Comment:
You can move `{configId}/client` to `@RequestMapping("api/docker/{configId}/client")`